<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>媽寶測驗大調查</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="sex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>性別:</string>
    </property>
    <widget class="QRadioButton" name="boy">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>男</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="girl">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>女</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="home">
       <property name="text">
        <string>住家裡</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="dormitory">
       <property name="text">
        <string>宿舍</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outside">
       <property name="text">
        <string>外面租</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="address">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>居住地:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="answer">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>91</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>同意此問卷</string>
    </property>
   </widget>
   <widget class="QListWidget" name="joblist">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>學生獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>魔法獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>奶油獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>老獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>好獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>比你獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>汽車維修獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上班獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>服務獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>獅獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>小獅獅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="job">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>職業:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>生活小日常:</string>
    </property>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽煮飯給我吃</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽叫我起床</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽載我去上學</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽會送東西給我</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽幫我洗衣服</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽幫我挑衣服</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽陪我去補習</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽給我錢錢</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽陪我睡覺</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>媽媽等我回家</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>送出</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="answer2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>31</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>195</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>性別</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="answer3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>111</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>居住</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="answer4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>71</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>職業</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="answer5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>201</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>測業結果</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
